### ISSUE TITLE
hmdb51 test file (before renamed)

### ISSUE THREAD

**Issue by ZealACMer** [youjiangxu/seqvlad-pytorch#7]
Thank you so much for your kind help. My question is do you have the test file before renamed by AllVideos_renamed.txt? This is very important for our experiment, thank you a lot for your kind help. Thank you again for all the helps you have provided, we deeply appreciate it. 
Reference:
https://github.com/ZealACMer/seqvlad-pytorch/tree/master/data/hmdb51_splits

**Comment by ZealACMer** [comment]
I want to fuse the hmbd51 score files provided in this repository with the score files obtained by Coviar(https://arxiv.org/pdf/1712.00636.pdf), however, I found the test files for these two methods are different. So I tried to change  the format of hmdb51_split1 test file in https://github.com/youjiangxu/seqvlad-pytorch/blob/master/data/hmdb51_splits/test_split1.txt to the format of the corresponding test file in https://github.com/chaoyuaw/pytorch-coviar/blob/master/data/datalists/hmdb51_split1_test.txt. When I finished this procedure, I retest the model of coviar by the new test file in accordance with SeqVLAD, then I tried to fuse these two npz score file(provided by your paper and Coviar, respectively) by late fusion(score fusion), but the result is lower than before fusion. Is there any problem with this procedure, can you give me some tips? Thank you very much. I am confused about this situation.

**Comment by ZealACMer** [comment]
In original paper of Coviar (https://arxiv.org/pdf/1712.00636.pdf), the experimental results show that the combination of Coviar with flow can further boost the classification performance. I believe your classification score file is good for fusion with Coviar, I am stucked here. Thank you so much for your kind help.

**Comment by youjiangxu** [comment]
Hi，
I think that you could try to fuse the two score files provided by SeqVLAD and Coviar with different weights because sometimes the weights are important to the final performance. And you could perform the grid search to find out the best weights?

Another tip is that I think you could double check the test split you transformed is correct. For example, when fusing these two score file, you could set the weight for your transformed test to 1 and the score file from Coviar to 0, to verify the performance of the transformed test split is the same as that of the original SeqVLAD score file.

**Comment by ZealACMer** [comment]
My idea is although the training lists of these two methods(SeqVLAD and Coviar) are slight different, which means the  two models are training on the slight different data(the class id is also different), as long as I change the test file to be the same, and make sure these two models are testing on the same data, the fused classfication performance won't hurt much. Am I right about this?
Thank you so much for your reply.

**Comment by youjiangxu** [comment]
I think that it would be better to make sure the classification score of SeqVLAD pluses on the same class id as Coviar. For example, if boxing's class id in SeqVLAD and Coviar are the 3th the 5th respectively, you should plus the 3th score value (after softmax) of SeqVLAD on the 5th score value (after softmax) of Coviar, but do not simply change the test list to the same name and the class id.

In addition, I think it is possible to do harm to the performance of the fused classification score due to the difference in the training data of the two methods. But it is hard to say how much influence it has.

**Comment by ZealACMer** [comment]
Thank you very much for your kind help. Thanks a lot.
